backend/de.metas.util/src/main/java/de/metas/util/async/Debouncer.java

```diff
@@ -193,6 +193,27 @@ public int getCurrentBufferSize()
 		}
 	}
 
+	public void processBufferSync()
+	{
+		synchronized (lock)
+		{
+			if (!buffer.isEmpty())
+			{
+				final ArrayList<T> itemsToConsume = new ArrayList<>(buffer);
```

Comment:
allthough it's aparently not required in this issue, i think the buffer should be cleared none the less. wdyt?